contrib/helm/harbor/templates/adminserver/adminserver-ss.yaml

```diff
@@ -56,6 +56,18 @@ spec:
           items:
           - key: secretKey
             path: key
+    {{- with .Values.adminserver.nodeSelector }}
+      nodeSelector:
+{{ toYaml . | indent 8 }}
```

Comment:
why not using `toYaml .Values.adminserver.nodeSelector | indent 8` but `with .Values.adminserver.nodeSelector` ?

Reply:
I followed the template from `helm create`.